docs/flows/protocol-management-flow/protocol-deployment-flow.md

```diff
@@ -0,0 +1,11 @@
+A sequence of the following families (in order):
```

Comment:
Did this previous include yield list deploying?
If it does, we might need to differentiate between "reference script deploying" for the _control_ scripts, versus for the authorized scripts. (We can't deploy the control script with an `authorizedScriptSTT`, because then the yielding scripts would authorize themselves -- it'd be a tautology!)